python/pfs/drp/stella/constructFiberFlatTask.py

```diff
@@ -142,3 +164,24 @@ def combine(self, cache, struct, outputId):
         self.write(cache.butler, flatExposure, outputId)
 
         return afwMath.binImage(flatExposure.image, self.config.binning)
+
+    def correctLowVarianceImage(self, maskedImage, minVar):
+        """Check the variance plane in the input image
+        for low variance values
+        and interpolate the variance if necessary.
+
+        Parameters
+        ----------
+        maskedImage : `lsst.afw.image.maskedImage.maskedImage.MaskedImageF`
+            the input maskedImage, whose variance plane is corrected.
+        detector : `lsst.afw.cameraGeom.detector.detector.Detector`
+            the instrument detector information.
+        minVarRN : float
+            the minimum variance, in units of read noise, below which the variance will
+            be interpolated.
```

Comment:
It's not in units of the read noise: that multiplication happens in the call, not in the function. This is an absolute value.

Reply:
Agreed. Fixed.